### PR TITLE
feat(helm): inspect readme

### LINKED ISSUE #2743: Fetch readme
Currently, the only way to look at the README file distributed with a chart is to look at it in git, or extract it out of the package tarball.

There needs to be a command like:
helm readme <chart>

which will fetch the package from the remote repo if its not already there, extract the readme, and cat it out to stdout.

### PR DESCRIPTION
**Pull request by f10et**
"helm readme" command allows to print the README.md file from a chart to stdout

closes #2743

### CODE REVIEW

**Comment by michelleN** [comment]
hey @scboffspring -- Thanks for your PR. It looks like very helpful functionality. What do you think about nesting this functionality under `helm inspect` in the command: `helm inspect readme` ?

**Comment by scboffspring** [comment]
Hey @michelleN,
Made total sense. Updated the PR to be under helm inspect as a 3rd option

File: cmd/helm/inspect.go [hunk 1]
```diff
@@ -184,16 +216,36 @@ func (i *inspectCmd) run() error {
...
-	if (i.output == valuesOnly || i.output == both) && chrt.Values != nil {
-		if i.output == both {
+	if (i.output == valuesOnly || i.output == all) && chrt.Values != nil {
+		if i.output == all {
 			fmt.Fprintln(i.out, "---")
 		}
 		fmt.Fprintln(i.out, chrt.Values.Raw)
 	}
 
+	if i.output == readmeOnly || i.output == all {
```
Comment (jascott1) [inline cmd/helm/inspect.go]: For charts without a readme file, if you don't specify `readme` subcommand you will still see 
```
---
No readme found in template
```
Please modify this to skip that output unless the `readme` subcommand is invoked. The other subcommands do not have a `not found` message so we could consider dropping it altogether.

**Review by jascott1 — CHANGES_REQUESTED** [review]
Looking good! Thanks for doing this one. I only found one issue.

**Comment by bacongobbler** [comment]
ping @scboffspring, are you still working on this PR?

**Comment by scboffspring** [comment]
Sorry, going through a few busy weeks.
Will try to finish it, and fix the conflicts hopefully tomorrow evening.

**Comment by jascott1** [comment]
Hi @scboffspring 

Thanks for the great work on the PR. Do you think you will have time to finish this up? 

Thanks,

Justin

File: cmd/helm/inspect.go (hunk 1 above)
Comment (scboffspring) [reply cmd/helm/inspect.go]: I will drop it out altogether

**Comment by scboffspring** [comment]
Hi @jascott1 

Sorry for the delay.

I have changed my pull request to remove the message when no readme is found in the chart

**Comment by scboffspring** [comment]
Hi @jascott1 just checking in if any more changes are needed

**Review by jascott1 — APPROVED** [review]

**Comment by jascott1** [comment]
Hi @scboffspring 

So sorry I let this one sit around and now we need to rebase. If you could, please rebase this PR and I will ping another core to take a look and hopefully approve it. Thanks!

**Comment by scboffspring** [comment]
Merged master and re-generated the documentation. 

You should be able to squash-merge this into master :)

**Comment by scboffspring** [comment]
@jascott1 just checking in, can this be merged ?

**Comment by bacongobbler** [comment]
I can take a look at this one today. Since it's a size/L it needs two maintainers to sign off. :)

**Review by bacongobbler — APPROVED** [review]
this looks great. Thanks!

**Comment by bacongobbler** [comment]
@scboffspring if you can rebase this PR we can go ahead and merge this one in :)

**Comment by scboffspring** [comment]
PR has been merged, it can be merged :)
@bacongobbler @jascott1

**Comment by bacongobbler** [comment]
CI failure's a flake that appears to be affecting most of our runs this morning. Merging!

**Pull request merged**